packages/Webkul/Installer/src/Console/Commands/DefaultUser.php

```diff
@@ -0,0 +1,455 @@
+        'kok_IN',
+        'ko_KR',
+        'ky_KG',
+        'lb_LU',
+        'lo_LA',
+        'lt_LT',
+        'lv_LV',
+        'mi_NZ',
+        'mk_MK',
+        'ml_IN',
+        'mn_MN',
+        'mn_Mong_CN',
+        'moh_CA',
+        'mr_IN',
+        'ms_BN',
+        'ms_MY',
+        'mt_MT',
+        'nb_NO',
+        'ne_NP',
+        'nl_BE',
+        'nl_NL',
+        'nn_NO',
+        'nso_ZA',
+        'oc_FR',
+        'or_IN',
+        'pa_IN',
+        'pl_PL',
+        'prs_AF',
+        'ps_AF',
+        'pt_BR',
+        'pt_PT',
+        'qut_GT',
+        'quz_BO',
+        'quz_EC',
+        'quz_PE',
+        'rm_CH',
+        'ro_RO',
+        'ru_RU',
+        'rw_RW',
+        'sah_RU',
+        'sa_IN',
+        'se_FI',
+        'se_NO',
+        'se_SE',
+        'si_LK',
+        'sk_SK',
+        'sl_SI',
+        'sma_NO',
+        'sma_SE',
+        'smj_NO',
+        'smj_SE',
+        'smn_FI',
+        'sms_FI',
+        'sq_AL',
+        'sr_Cyrl_BA',
+        'sr_Cyrl_CS',
+        'sr_Cyrl_ME',
+        'sr_Cyrl_RS',
+        'sr_Latn_BA',
+        'sr_Latn_CS',
+        'sr_Latn_ME',
+        'sr_Latn_RS',
+        'sv_FI',
+        'sv_SE',
+        'sw_KE',
+        'syr_SY',
+        'ta_IN',
+        'te_IN',
+        'tg_Cyrl_TJ',
+        'th_TH',
+        'tk_TM',
+        'tn_ZA',
+        'tr_TR',
+        'tt_RU',
+        'tzm_Latn_DZ',
+        'ug_CN',
+        'uk_UA',
+        'ur_PK',
+        'uz_Cyrl_UZ',
+        'uz_Latn_UZ',
+        'vi_VN',
+        'wo_SN',
+        'xh_ZA',
+        'yo_NG',
+        'zh_CN',
+        'zh_HK',
+        'zh_MO',
+        'zh_SG',
+        'zh_TW',
+        'zu_ZA',
+    ];
+
+    /**
+     * Create UnoPim user.
+     */
+    public function handle()
+    {
+        Log::info('User create command has started');
+
+        $userName = $this->option('name');
+        $isAdmin = $this->option('admin');
+
+        if ($userName && ! preg_match('/^[a-zA-Z0-9\s]+$/', $userName)) {
+            $this->warn('The name can only accept alphanumeric characters and spaces.');
+            Log::warning('The name can only accept alphanumeric characters and spaces.');
+            $userName = null;
+        }
+
+        $userName = $userName ?: text(
+            label: 'Set the Name for User',
+            default: $isAdmin ? 'Admin' : 'User',
+            required: true,
+            validate: fn (string $value) => match (true) {
+                ! preg_match('/^[a-zA-Z0-9\s]+$/', $value) => 'The name can only accept alphanumeric characters and spaces.',
+                default                                    => null
+            }
+        );
+
+        $userEmail = $this->option('email');
+        $existingUserEmails = DB::table('admins')->pluck('email')->toArray();
+
+        if ($userEmail && ! filter_var($userEmail, FILTER_VALIDATE_EMAIL)) {
+            $this->warn('The provided email is invalid, kindly enter a valid email address.');
```

Comment:
create a common function for $this->warn and log warning used multiple times with the same message.

Reply:
added the function